spec/decorators/odk/form_decorator_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+# Form decorator spec
+module Odk
+  describe FormDecorator, :odk, :reset_factory_sequences do
+    describe "needs_manifest?", :odk do
```

Comment:
Since the outer block has `:odk` on it already, the inner one doesn't need it.

Also, why did you add `:reset_factory_sequences`?

Reply:
`reset_factory_sequences` reloads the factories